git-guide/git-topic-1/index.html

```diff
@@ -56,124 +56,224 @@
+
       </div>
-    </div>
-    <div class="pusher">
-      <div class="ui basic container segment">
-       
-         
-  
+     
+      <div  class="ui tab segment lab" data-tab="02">
+        <h1>Terminal</h1>
+<h2>What is a terminal?</h2>
+<p>The terminal is an interface in which you can type and execute text based commands.</p>
 
-         
-         
-  
-    <section class="ui attached segment">
-      <a class="ui right ribbon label">talks</a>
-      <div class="ui centered four doubling stackable cards">
-  
-     <a class="ui raised link card" href="talk-1-git/git.pdf" target="_blank">
-  
+<p float="left"><img src="img/terminal.png" alt="A Terminal View"></p>
 
-      <div class="center aligned content">
-        <div class="right floated meta"><i class="huge object group icon"></i></div>
-        <div class="header"> Git</div>
-      </div>
+<p>Another way of describing an interface is how something is interacted with</p>
+<p>Some examples are<p>
+<ul>
+<li>A Steering wheel in a car</li>
+<li>A Control on a Games Console</li>
+<li>A Keyboard on a computer</li>
+<li>GUI - Graphical User Interface</li>
+</ul>
 
-      <div class="image">
-        <img class="ui bordered rounded image" src="talk-1-git/git.png">
-      </div>
+<p>Can you think of any other examples?</p>
+
+<h2>Alternative Names for Terminal</h2>
+  <ul>
+    <li>Console</li>
+    <li>Command Line</li>
+    <li>Command Prompt</li>
+    <li>Shell</li>
+  </ul>
+
+<h2>How to get access to your terminal</h2>
+
+<p> This depends on the operating system you are using</p>
+<ul>
+<li>On Mac - use the SearchLight Function</li>
+<li>On Linux - use the Search Program Function</li>
+</ul>
 
-      <div class="center aligned content">
-        <div class="description">
-          <p>A visual introduction</p>
 
-        </div>
       </div>
+     
+      <div  class="ui tab segment lab" data-tab="03">
+        <h1>Terminal Commands</h1>
+        <h2>How to navigate around folders in the terminal</h2>
+        <p>You will see <username>@<computer> foldername  .This is your folder path.</p>
+        <p>This tells you what folder you are in.</p>
+        <p>To see what is the folder type ‘ls -al’ and hit enter. (see the image below for an example)</p>
 
-      
 
-    </a>
+<p float="left"><img src="img/folder.png" alt="A Folder View"></p>
+
+<p>
+This will show you a list of all files and folders inside the folder you are currently in. To move to a different folder inside the folder you are currently in type: cd <folder> (see the image below for an example. cd=Change Directory
+</p>
+
+<p float="left"><img src="img/sample.png" alt="Change Folder"></p>
+
+
+<p>Using these you commands you should be able to get to the folder where your My-First-Website files are stored. This is where we want to initialise (create) our local git repository (project).</p>
+
+<p>NOTE: If you go into a folder that you want to get out of type ‘cd ..’ and hit enter. This brings you back up one folder.</p>
```

Comment:
May also be worth noting that `.` means the current folder and `..` means the outer folder.